src/Microsoft.SqlTools.ServiceLayer/LanguageServices/AutoCompleteHelper.cs

```diff
@@ -728,5 +728,137 @@ internal static SignatureHelp ConvertMethodHelpTextListToSignatureHelp(List<Babe
 
             return help;
         }
+
+        /// <summary>
+        /// Give suggestions for sql star expansion. 
+        /// </summary>
+        /// <param name="scriptDocumentInfo">Document info containing the current cursor position</param>
+        /// <returns>Completion item array containing the expanded star suggestion</returns>
+        public static CompletionItem[] ExpandSqlStarExpression(ScriptDocumentInfo scriptDocumentInfo)
+        {
+            //Fetching the star expression node in sql script.
+            Microsoft.SqlServer.Management.SqlParser.SqlCodeDom.SqlSelectStarExpression selectStarExpression = AutoCompleteHelper.isSelectStarStatement(scriptDocumentInfo.ScriptParseInfo.ParseResult.Script, scriptDocumentInfo);

```

Comment:
Would suggest a `using` statement here (`using Microsoft.SqlServer.Management.SqlParser.SqlCodeDom;` to shorten this and the other variable names here